### PR TITLE
#132: Update latest release

### LINKED ISSUE #132: Update Latest/Next Releases on support page
* **Browser**:

The “Latest Release” and “Next Release” columns in the table on https://adoptium.net/support.html need to be updated.

### PR DESCRIPTION
**Pull request by jeffalder**
Fixes #132 

##### Checklist

- [x] `npm test` passes
- [x] documentation is changed or added (if applicable)
- [x] permission has been obtained to add new logo (if applicable)
- [x] contribution guidelines followed [here](https://github.com/adoptium/adoptium.net/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by jeffalder** [comment]
Screen Shot 2021-08-02 at 16 24 24

File: src/handlebars/support.handlebars [hunk 1]
```diff
@@ -68,49 +68,37 @@
...
-              <strong>jdk8u282-b08<br/>
-              19th Jan 2021</strong>
+              <strong>jdk8u302-b08<br/>
+              29th Jul 2021</strong>
           </td>
           <td>
-              <strong>jdk8u292<br/>
-              20th Apr 2021</strong>
+              <strong>jdk8u312<br/>
+              Sep 2021</strong>
```
Comment (karianna) [inline src/handlebars/support.handlebars]: Oct

File: src/handlebars/support.handlebars [hunk 2]
```diff
@@ -68,49 +68,37 @@
...
-              19th Jan 2021</strong>
-
+            <strong>jdk-11.0.12+7<br/>
+              1st Aug 2021</strong>
           </td>
           <td>
-            <strong>jdk-11.0.11<br/>
-              20th Apr 2021</strong>
+            <strong>jdk-11.0.13<br/>
+              Sep 2021</strong>
```
Comment (karianna) [inline src/handlebars/support.handlebars]: Oct

**Review by karianna — CHANGES_REQUESTED** [review]
2 small changes

**Review by karianna — APPROVED** [review]

**Comment by jeffalder** [comment]
@karianna Sorry for not turning the changes around quickly enough!

**Comment by karianna** [comment]
Oh not at all - we're all volunteers here and I was just trying to avoid having to use my less than stellar git skills as I had a potentially clashing PR :-)

**Pull request merged**